### PR TITLE
shopfloor_mobile_base: make home customizable

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Comment by JuMiSanAr** [comment]
Updated so that the event on successful login now redirects to the page set as home.

File: shopfloor_mobile_base/static/wms/src/router.js [hunk 1]
```diff
@@ -32,9 +26,16 @@ const routes = [
     // TODO Fix this it needs to be the last route, but I think it is not anymore with the dynamic one added.
     // { path: '*', component: NotFound },
 ];
+const default_home_route = {
+    path: "/",
+    component: HomePage,
+    name: "home",
+    meta: {requiresAuth: true, requiresProfile: true},
```
Comment (simahawk) [inline shopfloor_mobile_base/static/wms/src/router.js]: @JuMiSanAr you must set `isHomeRoute` here otherwise your filter on login won't work

**Review by JuMiSanAr — APPROVED** [review]

**Comment by simahawk** [comment]
this is not over: we have to take care of all the places where the route "home" is used directly. EG: https://github.com/OCA/wms/blob/f4a3d433ee65d279726902eb762300cbd0ffbba0/shopfloor_mobile_base/static/wms/src/settings/profile.js#L54

**Comment by simahawk** [comment]
> this is not over: we have to take care of all the places where the route "home" is used directly. EG:
> 
> https://github.com/OCA/wms/blob/f4a3d433ee65d279726902eb762300cbd0ffbba0/shopfloor_mobile_base/static/wms/src/settings/profile.js#L54

fixed

**Comment by JuMiSanAr** [comment]
Ended up using a redirect in the router so that whenever the user lands on "/", they're redirected to the home route of their choice.

**Pull request merged**